site/zenodo_rdm/migrator/stream_definition.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""Migrator stream definitions."""
+
+from invenio_rdm_migrator.extract import Extract
+from invenio_rdm_migrator.streams import StreamDefinition
+from invenio_rdm_migrator.transform import Transform
+
+
+class ZenodoDirectLoadStreamDefition(StreamDefinition):
+    """Zenodo to RDM direct loading class i.e no ET steps."""
+
+    class NoExtract(Extract):
```

Comment:
You can put those two calsses as root (not inner calsses). Theren there would be no need to overwrite and could use `StreamDefinition` directly (i.e. simplifying the __main__)

Reply:
I didn't want that just because the DirectLoad makes sense only if you pass these dummy internal classes...